runtime/src/gpu/nvidia/gpu-nvidia.c

```diff
@@ -64,8 +64,8 @@ static bool chpl_gpu_has_context(void) {
   }
 }
 
-static void chpl_gpu_switch_context(int deviceId) {
-  CUcontext next_context = chpl_gpu_primary_ctx[deviceId];
+static void switch_context(int dev_id) {
```

Comment:
> Ok, I suspect the answer is "it's chaos, we're not consistent, and there's bigger fish to fry than naming conventions", but it would be nice if I knew what I should be trying to do going forward so I could avoid adding to the chaos.

If me making the call on this helps, let's go with `snake_case` in the GPU code in the runtime. To me the only exception should be if we're naming a runtime wrapper of a an exported Chapel proc that has camelCase name. e.g. `chpl_gpu_fooBar` is ok if it wraps around `export proc fooBar`.

I know the runtime is chaos with capitalization, but I think snake_case is more common. It certainly is the case for the GPU code. Of course you can say that "you showing your own code as precedence is not fair" :)